llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -1610,6 +1610,8 @@ bool RISCVFrameLowering::assignCalleeSavedSpillSlots(
         int FrameIdx = MFI.CreateFixedSpillStackObject(Size, Offset);
         assert(FrameIdx < 0);
         CS.setFrameIdx(FrameIdx);
+        if (RISCVRegisterInfo::isRVVRegClass(RC))
```

Comment:
There aren't any vector registers in `FixedCSRFIMap` so I don't think this can happen.

Reply:
addressed